docker/odklite/Dockerfile

```diff
@@ -73,6 +73,9 @@ RUN wget -nv https://github.com/lihaoyi/Ammonite/releases/download/$AMMONITE_VER
 # Install RDF/XML validation script
 COPY scripts/check-rdfxml.sh /tools/check-rdfxml
 
+# Force Git to accept working on a repository owned by someone else
+RUN echo "[safe]\n    directory = /work" >> /etc/gitconfig
```

Comment:
What exactly does this do? Never seen this before!

Reply:
Any security risks you can think of?

Reply:
None. The repository _does_ belong to the calling user anyway, only the top-level directory to which it is bound does not.

Reply:
Thank you!